src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -663,6 +691,12 @@ bool Flag_CCSN( const int i, const int j, const int k, const int lv, const int P
 
    bool Flag = false;
 
+   if (  ( CCSN_Prob == Core_Collapse )  &&  !CCSN_Is_PostBounce  )
+   {
+      Flag |= Flag_CoreCollapse( i, j, k, lv, PID, Threshold );
+      if ( Flag )    return Flag;
```

Comment:
What if we start from collapse and then pass through core bounce. Will `Flag_Lightbubl` triggered after core bounce?

Reply:
https://github.com/Inhyeok-Song/gamer-fork/blob/7177cc93014299d5ccfcae62ba57410b3e0ea252/src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp#L615-L627

That part is here. We can start from core collapse and if the core bounce is detected, the deleptonization will turn off and the lightbulb will turn on.